src/main/resources/license-data.json

```diff
@@ -607,6 +607,16 @@
       "license": "MIT",
       "licenseSource": "data"
     },
+    "com.mchange:c3p0": {
```

Comment:
Where does c3p0 come from? We already have HikariCP as a connection pool

Reply:
we have it in the platform dependencies: https://github.com/xebialabs/xl-platform/pull/858/files#diff-23ac65326c33b66912a09c7ad169aa8eR116
and there is an exclusion in XLR: https://github.com/xebialabs/xl-release/blob/9b592c14ed34cc62eb555dd83b6514d66f588141/core/xlr-server/build.gradle#L47

it looks like a forgotten dependency. Shall we get rid of it?